tests/components/dynalite/test_const.py

```diff
@@ -0,0 +1,44 @@
+"""Check that the consts in the component / HA are the same as in the library."""
+from dynalite_devices_lib import const as dyn_const
+
+from homeassistant.components import dynalite
+import homeassistant.const as ha_const
+
+
+def test_consts():
+    """Verify that the consts defined in HA and the component are the same as the ones in the library."""
+    assert ha_const.CONF_HOST == dyn_const.CONF_HOST
```

Comment:
Why are we testing this?

Reply:
the integration assumes that the CONF consts defined in the integration are the same as the ones in the library.
if in a future version of the library, one of these CONF variables change (e.g. instead of "port" it will be "port2"), the library tests will still work (they all use the same const.py file), and the integration tests would also still work (since they mock the library and probably don't really use most CONF), but together this will fail. 
Adding this test ensures that upgrading to a future version of the library will only pass the tests if the CONF values were not changed. If they do, I will have to write some translator to upgrade in a non-breaking way to the library before upgrading to that version of the library.
This will ensure that I don't push a change that breaks this behavior unknowingly

Reply:
We shouldn't couple the two APIs like this. Please decouple them.

Reply:
that makes sense. it will make the code in bridge.py more complicated as i will need to rebuild the schema, but with the values from the library instead of the ones from HA. 
will get working on that.

Reply:
ok. i created the mapping and removed the test.
thinking about refactoring the library to get a class instead of a dict to make it safer